.travis.yml

```diff
@@ -1,14 +1,28 @@
 language: perl
+
 perl:
     - "5.28"
     - "5.26"
     - "5.24"
     - "5.22"
     - "5.20"
     - "5.16"
+
 before_install:
     - eval $(curl https://travis-perl.github.io/init) --auto
-    - local-lib
-    - git clone --depth=1 --branch=develop https://github.com/zonemaster/zonemaster-ldns.git
-    - git clone --depth=1 --branch=develop https://github.com/zonemaster/zonemaster-engine.git
-    - cpan-install --deps Devel::CheckLib Module::Install Module::Install::XSUtil ./zonemaster-ldns ./zonemaster-engine
+    - sudo apt-get install -y libidn11-dev
+    - cpan-install --deps Devel::CheckLib Module::Install Module::Install::XSUtil
+    - git clone --depth=1 --branch=$TRAVIS_BRANCH https://github.com/zonemaster/zonemaster-ldns.git
+    - ( cd zonemaster-ldns ; cpanm --notest --configure-args="--no-ed25519" . )
+    - git clone --depth=1 --branch=$TRAVIS_BRANCH https://github.com/zonemaster/zonemaster-engine.git
+    - ( cd zonemaster-engine ; cpanm --notest ./zonemaster-engine )
```

Comment:
Shouldn't it be 
```
cd zonemaster-engine ; cpanm --notest .
```
or even
```
cd zonemaster-engine && cpanm --notest .
```

Reply:
I guess && would be better, but I'm not sure if/how I need to escape it.